### PR TITLE
Applicants project grants mobile hero

### PR DESCRIPTION
**Pull request by nhsz**
This PR adds the hero section for `/applicants/project-grants` (mobile version)

PREVIEW: https://deploy-preview-115--thirsty-meitner-cae269.netlify.app/applicants/project-grants

- custom `theme.ts` added for `ChakraProvider`
- `/applicants/project-grants` page created
- `/components/UI` base components created, to be used on different pages (`Layout`, `NavMobile`, `PageHeading`, `PageText`, `PlaceholderImage`)

> **NOTE:** `components/UI` will be refactored later as needed, on separated directories

### CODE REVIEW

File: src/components/UI/NavMobile.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+import { Box, Flex, IconButton, Menu, MenuButton } from '@chakra-ui/react';
+import Image from 'next/image';
+import { FC } from 'react';
+import logoSrc from '../../public/esp-logo.svg';
+
+const HamburgerIcon = createIcon({
+  displayName: 'HamburgerIcon',
+  viewBox: '0 0 32 25',
+  path: <path fill='#F85858' d='M0 0h32v3H0zM0 11h32v3H0zM0 22h32v3H0z' />
+});
```
Comment (pettinarip) [inline src/components/UI/NavMobile.tsx:11]: Probably you have already thought about this but would be good to have all the icons grouped in their own folder. A refactor we can implement in next PRs when you have more icons.

File: src/components/UI/PageHeading.tsx [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import { Heading } from '@chakra-ui/react';
+import { FC } from 'react';
+
+export const PageHeading: FC = ({ children }) => {
+  return (
+    <Heading as='h1' color='brand.heading' fontSize='h1' fontWeight={200} lineHeight='48px' mb={4}>
```
Comment (pettinarip) [inline src/components/UI/PageHeading.tsx:6]: For some reason, this heading is not using the font "Libre Franklin" that you have declared in the theme.
image

**Review by pettinarip — APPROVED** [review]
LGTM @nhsz gj. I've just added a couple of minor comments. Take a look at the font, for some reason its not displaying correctly.

File: src/components/UI/NavMobile.tsx (hunk 1 above)
Comment (nhsz) [reply src/components/UI/NavMobile.tsx:11]: Agree, I've moved `<HamburguerIcon />` to `/components/UI/icons`

File: src/components/UI/PageHeading.tsx (hunk 2 above)
Comment (nhsz) [reply src/components/UI/PageHeading.tsx:6]: It seems to be working fine on my browser, it's using Libre Franklin

Screen Shot 2021-12-27 at 21 28 55

Comment (pettinarip) [reply src/components/UI/PageHeading.tsx:6]: Oh ok nice then. I was comparing it with the design where we might be using a smaller font weight:
image

Comment (pettinarip) [reply src/components/UI/PageHeading.tsx:6]: Yes, it should be font-weight 200

Comment (nhsz) [reply src/components/UI/PageHeading.tsx:6]: The `F` is glitched in the design on figma, all the text should be font-weight 200 as you said

**Pull request merged**